Documentation/Storage-Configuration/requirements.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Requirements
+---
+
+## Software
+
+Check out the [pre-requisite](../Getting-Started/Prerequisites/prerequisites.md) section to get started.
+
+### CPU Architecture
+
+We only support `amd64 / x86_64` and `arm64` architecture.
+
+### Kubernetes
```

Comment:
Let's move our supported K8S versions and distro table to the Prerequisites page, and also rename the "Minimum Version" section to "Minimum Kubernetes Version"